inter/event_serializer.go

```diff
@@ -17,8 +17,11 @@ import (
 var (
 	ErrSerMalformedEvent = errors.New("serialization of malformed event")
 	ErrTooLowEpoch       = errors.New("serialization of events with epoch<256 and version=0 is unsupported")
+	ErrUnknownVersion    = errors.New("unknown serialization version")
```

Comment:
why you made `ErrUnknownVersion` public ?  It is used only within `inter` package here:
```
	if version > MaxSerializationVersion {
		return ErrUnknownVersion
	}
```

Reply:
ditto  `ErrSerMalformedEvent` and `ErrTooLowEpoch`

Reply:
The most `ErrX` variables in this repo are public but never used outside their package. If we were to refactor it, we may need a separate PR